package/prudynt-t/files/configs/tplink_tapo_c100v5

```diff
@@ -0,0 +1,3 @@
+sensor: {
+        fps: 15;
```

Comment:
I think this sensor can do 30fps...

Reply:
surprisingly, they have both entries in the header, both for mipi_2lane.
<https://github.com/sstepansky/ingenic-sdk/blob/7998f234adade2d7c2b3af75622582e88b133639/3.10/sensor-src/t23/sc2336p.c#L8-L10>

Reply:
Nice.
I will change the max in the source to 30 and see what happens!

Reply:
It works on 30 fps! I already changed the sensor source.